src/main/telemetry/ghst.c

```diff
@@ -143,7 +144,7 @@ void ghstFrameGpsSecondaryTelemetry(sbuf_t *dst)
     sbufWriteU16(dst, gpsSol.groundCourse);     // degrees * 10
     sbufWriteU8(dst, gpsSol.numSat);
 	
-    sbufWriteU16(dst, (uint16_t) (GPS_distanceToHome / 10));    // use units of 10m to increase range of U16 to 655.36km
```

Comment:
It's defined as `uint16_t` before but does removing the cast after divide allow it to be another type now?

Reply:
The second param of `sbufWriteU16` is `uint16_t`
`GPS_distanceToHome` is `uint16_t`

Am I missing something?

Reply:
No, just an observation. If `sbufWriteU16` has second param declared as `uint16_t` it should be no problem.

Reply:
Ok, resolved.